lib/sqlalchemy/orm/scoping.py

```diff
@@ -468,7 +468,8 @@ def begin_nested(self) -> SessionTransaction:
 
             :ref:`pysqlite_serializable` - special workarounds required
             with the SQLite driver in order for SAVEPOINT to work
-            correctly.
+            correctly. The same holds true for the aiosqlite driver
+            :ref:`aiosqlite_serializable`.
 
 
         """  # noqa: E501
```

Comment:
**Federico Caselli** (CaselIT) wrote:

Done

View this in Gerrit at https://gerrit.sqlalchemy.org/c/sqlalchemy/sqlalchemy/+/4907